source/jormungandr/jormungandr/scenarios/new_default.py

```diff
@@ -1258,37 +1258,34 @@ def isochrone_common(isochrone, request, instance, journey_req):
     journey_req.streetnetwork_params.destination_mode = isochrone.destination_modes[0]
 
 
-def is_olymbic_poi(pt_object, instance):
+def is_olympic_poi(pt_object, instance):
     if pt_object.embedded_type == type_pb2.POI:
         if not (hasattr(pt_object.poi, 'properties') and pt_object.poi.properties):
             return False
-    if pt_object.embedded_type == type_pb2.POI:
-        olympic_site = next(
-            (
-                p
-                for p in pt_object.poi.properties
-                if p.type == instance.olympics_forbidden_uris.poi_property_key
-                and p.value == instance.olympics_forbidden_uris.poi_property_value
-            ),
-            None,
-        )
-        if olympic_site:
-            return True
+    olympic_site = next(
```

Comment:
Are you sure the indentation is correct here? This is the same level than `if pt_object.embedded_type == type_pb2.POI:` which means at this point, it might not be a POI. Since the function is called `is_olympic_poi`, something seems off (either the name of the function or the code in the function).